python/paddle/fluid/tests/unittests/mkldnn/test_elementwise_div_mkldnn_op.py

```diff
@@ -111,6 +111,45 @@ def test_check_grad_ignore_x(self):
         pass
 
 
+class TestMKLDNNElementwiseDivOpZeroDim(TestMKLDNNElementwiseDivOp):
```

Comment:
whether it need to check backward in Div/Mul/Sub OP?

Reply:
Hi, it seems original div/mul/sub cases with broadcast cannot pass with oneDNN. So I skip them temprarily. I'll check the broadcast in the next patch:)

Reply:
OK, I seed grad is checked when no broadcast